doc/tutorials.rst

```diff
@@ -20,37 +60,119 @@ plugins, and devices, via simple and easy-to-follow examples.
 
 .. customgalleryitem::
     :tooltip: Use quantum machine learning to tune a beamsplitter.
-    :figure: ../examples/figures/gaussian_transformation.png
+    :figure: ../examples/figures/gaussiancircuit.png
     :description: :ref:`gaussian_transformation`
 
+:html:`<div style='clear:both'></div>`
+
+Classical interfaces
+~~~~~~~~~~~~~~~~~~~~~
+
+The following tutorials take an in-depth look at one and the same Ising model tutorial, implemented across different interfaces offered by PennyLane. 
```

Comment:
```suggestion
The following tutorials demonstrate how to optimise the energy of a simple Ising model using different interfaces offered by PennyLane. 
```
I was very confused as to where the Ising model tutorial was mentioned before. Ok to change?